quarkchain/tools/quick_miner_tool.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+clear
+echo "Step 1: Enter or generate your coinbase address!"
+
+miner_tools_path="$(pwd -P)/quarkchain/tools/miner_address.py"
+pypy3 $miner_tools_path
+
+echo -n "Enter your coinbase address again!"   
+echo -n "Paste your address:"                
+read  address                                   
```

Comment:
ADDRESS

Reply:
The duplicated address input has been removed. Besides, I have revised the code according to the comments